### PR TITLE
Fixed WaterBuff error for all hover units!

### PR DESCRIPTION
**Pull request by JJ173**
PR will need testing and reviewing accordingly. Added the class to the base `nomadsunits.lua` file so that should run just fine. Please check before merging. Everything should be up to snuff.

### CODE REVIEW

**Comment by JJ173** [comment]
Fixes Issue: #119

File: lua/nomadsunits.lua [hunk 1]
```diff
@@ -341,6 +341,22 @@ NExperimentalHoverLandUnit = Class(NHoverLandUnit) {
     end,
 }
 
+SlowHoverLandUnit = Class(HoverLandUnit) {
```
Comment (CookieNoob) [inline lua/nomadsunits.lua]: no need to add this function (its in defaultunits, i.e. for all factions) for nomads specifically again.

File: units/INU1002/INU1002_Script.lua [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+-- t1 land scout
+
+local AddAnchorAbilty = import('/lua/nomadsutils.lua').AddAnchorAbilty
+local NHoverLandUnit = import('/lua/nomadsunits.lua').NHoverLandUnit
+local DarkMatterWeapon1 = import('/lua/nomadsweapons.lua').DarkMatterWeapon1
+local SlowHover = import('/lua/nomadsunits.lua').SlowHoverLandUnit
+
+NHoverLandUnit = AddAnchorAbilty(NHoverLandUnit)
+
+INU1002 = Class(NHoverLandUnit) {
```
Comment (CookieNoob) [inline units/INU1002/INU1002_Script.lua]: missing class SlowHover

```INU1002 = Class(NHoverLandUnit, SlowHover) {```

File: units/INU1002/INU1002_Script.lua [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
-}
-
-TypeClass = INU1002
\ No newline at end of file
+-- t1 land scout
+
+local AddAnchorAbilty = import('/lua/nomadsutils.lua').AddAnchorAbilty
+local NHoverLandUnit = import('/lua/nomadsunits.lua').NHoverLandUnit
+local DarkMatterWeapon1 = import('/lua/nomadsweapons.lua').DarkMatterWeapon1
+local SlowHover = import('/lua/nomadsunits.lua').SlowHoverLandUnit
```
Comment (CookieNoob) [inline units/INU1002/INU1002_Script.lua]: use the function from default units

**Pull request merged**